src/main/java/com/Saurabh/GitHubAiAPI/StringManipulation.java

```diff
@@ -45,4 +49,12 @@ public static String findVowels(String input) {
```

Comment:
The regular expression '[^a-zA-Z]' will remove all non-alphabetic characters. If the intention is to check for palindrome considering only alphabets, this is fine. However, if the palindrome check should consider alphanumeric characters, the regular expression needs to be updated.